samples/CookieSharing/README.md

```diff
@@ -8,4 +8,12 @@ The sample illustrates cookie sharing across three apps that use cookie authenti
 | CookieSharing.WithIdentity.Core         | ASP.NET Core 2.0 MVC app with ASP.NET Core Identity |
 | CookieSharing.WithIdentity.NETFramework | ASP.NET Framework 4.6.1 MVC app with ASP.NET Identity |
 
+Instructions:
+
```

Comment:
So isn't it a bit wonky that you can create the same user in all 3 apps?  I'd imagine this sample would be more realistic if all 3 apps were sharing the same database, maybe you can only create users in the identity apps, but its a bit weird for the example flow to ask you to create the same user in multiple apps.  if its too involved to get the databases shared in this PR then just file an issue for the future, and I'd at least drop the explicit steps to create duplicate users, and just demonstrate that you can create the user in one app, and use the cookie in the other apps without having them sign out/recreate users.

Reply:
I agree, but it's in-memory dB as far as the Core projects are concerned. That's why it turned out that way. We do all (or most ... haven't surveyed every one) of the doc samples with Identity using the in-memory dB approach, and IIRC it's a standing order to do so for all future samples. I think the goal is to simplify sample use, since most of them have little to do with the actual underlying dB and don't really want to push one brand. It just makes our samples super-easy to setup and run xplat and xIDE.

Reply:
That's fine for stand alone samples, but this repo is meant for demonstrate more realistic cookie/identity sharing.  We should just use an app.db or other EF shared database and demonstrate how to share identity db's between core and old as well.  We can put a caveat/commented out code for what modifications (if any) are needed for xplat (how to switch to in memory)

Reply:
But I'm basically trying to shoehorn two topics into these 3 apps.

Cookie sharing is what you've currently focused on.
Identity db sharing involves cookie sharing and db sharing, which is where I'm going with this.

Reply:
If you think its better to keep these two separate.

I would just do CookieSharing without identity.  CookieSharing.App1, CookieSharing.App2

And move the IdentityCore + IdentityNetFramework into its own thing Identity.Interop.Core, Identity.Interop.NetFramework

Reply:
It sounds fine to me ... and easier to explain in sample instructions the way you propose to go.

What I suggest tho is perhaps we:
* Merge the current sample into the docs repo as we were originally going to do
* Open a new docs issue to refactor that topic (and kill the samp over there) when the new sample setup is ready to link over here.

The reason I float this is that the PR topic is ready and the PR sample works. The current topic is wrong ... the live sample is broken. This way, we get live with something that at least works now. It also closes a priority issue for us, and this work over here can take as long as it needs.

Reply:
If we are agreed on the long term final results, I'm happy with whatever works best for you in the short term :)

Reply:
The last ? is where do you need me to leave this for you to pick it up. Should I go ahead with that last item: Get the netfx project into one folder, drop it's solution file, add the netfx project to the AuthSamples solution file. I guess leave the naming alone for now.

... or do you want me to just stop now?

Reply:
Yeah don't worry about the naming, if you can just put everything into the existing sln and make sure it all still runs/works, that'd be great

Reply:
kk ... will do.